applied-data-science-using-python.md

```diff
@@ -0,0 +1,20 @@
+Title | Offered By | Course Link
```

Comment:
- Add proper title and description at the beginning of the file
- Add metadata at the top

Reply:
@TACrafter please make these changes
Refer to exampleFile for more information